components/user/avatar-options.tsx

```diff
@@ -0,0 +1,66 @@
+import { Button, MenuItem, OverflowMenu } from "@ui-kitten/components";
+import Feather from "@expo/vector-icons/Feather";
+import { useState } from "react";
+
+import { Text } from "react-native";
+
+interface AvatarOptionProps {
+    openImageLibrary: () => Promise<void>;
+    openCamera: () => Promise<void>;
+    handleRemovePhoto: () => void;
+    isLoading: boolean;
+}
+
+function AvatarOptions(props: Readonly<AvatarOptionProps>) {
+    const [visible, setVisible] = useState(false);
+    const { openCamera, openImageLibrary, handleRemovePhoto, isLoading } =
+        props;
+    const toggleOptionsButtone = () => (
+        <Button
+            onPress={handleSetVisible}
+            disabled={isLoading}
+            status="basic"
+            style={{ paddingHorizontal: 0, paddingVertical: 0 }}
+        >
+            <Text>
+                <Feather name="edit" size={16} color="black" /> Edit
+            </Text>
+        </Button>
+    );
```

Comment:
convert to named function, also naay extra e sa last HAHAHAH Buttone

```tsx
function toggleOptionsButton() {
return (
        <Button
            onPress={handleSetVisible}
            disabled={isLoading}
            status="basic"
            style={{ paddingHorizontal: 0, paddingVertical: 0 }}  // Also can we use className here instead of styles? 
            // Maybe this would work? p-0
        >
            <Text>
                <Feather name="edit" size={16} color="black" /> Edit
            </Text>
        </Button>
    );
}
```

Reply:
HAHAHAHHA gegege

Reply:
btw just tried it, dili muwork the same ang 
`className="p-0"` for styling